SEP-0013.md

```diff
@@ -0,0 +1,198 @@
+the list does not have to be exhaustive. We envision that it can be appended to over time. But
+once an attribute name is addedit should preferably not be changed so that users can rely on a
+stable, backwards compatible API.
+
+For now, this API does not include WCS metadata and should not be linked to a specific file format,
+e.g. FITS.
+
+
+# SunPy Metadata API
+Below is a class definition of an Metadata Abstract Base Class (ABC) for solar physics.
+
+```python
+
+class MetaABC():
+    ##### Instrument info #####
+
+    @abc.abstractproperty
+    def observatory(self):
+        """
+        Name of the observatory
+        """
+
+    @abc.abstractproperty
+    def instrument(self):
+        """
+        Name of the instrument
+        """
+
+    @abc.abstractproperty
+    def detector(self):
+        """
+        Name of the detector
+        """
+
+    ##### Data Info ####
+
+    @abc.abstractproperty
+    def data_processing_level(self):
+        """
+        The level to which the data has been processed.
+        """
+
+    @abc.abstractproperty
+    def data_version(self):
+        """
+        The data version.
+        """
+
+    ##### Observer Coordinate Info #####
+
+    @abc.abstractproperty
+    def observer_location(self):
+        """
+        Coordinate of observatory location.
+        """
+
+    def observer_radial_velocity(self):
+        """
+        Velocity of observer in direction of source.
+        """
+
+    @abc.abstractproperty
+    def celestial_frame(self):
+        """
+        The celestial coordinate frame of the image axes.
+        """
+
+    @abc.abstractproperty
+    def distance_to_sun(self):
+        """
+        Distance to Sun center from observatory.
+        """
+
+    @abc.abstractproperty
+    def light_travel_time(self):
+        """
+        Light travel time from emission source to observer.
+        """
+
+    @abc.abstractproperty
+    def light_travel_time_to_earth(self):
+        """
+        Light travel time from observer to Earth.
+        """
+
+    ##### Time Info #####
+
+    @abc.abstractproperty
+    def date_reference(self):
+        """
+        The base time from which time axis values are measured.
+        """
+
+    @abc.abstractproperty
+    def date_start(self):
+        """
+        Time at which the observation(s) were started.
+        """
+
+    @abc.abstractproperty
+    def date_average(self):
+        """
+        The mean/representative time of the observation(s).
+        """
+
+    @abc.abstractproperty
+    def date_end(self):
+        """
+        The time at which the observation(s) were ended.
+        """
+
+    @abc.abstractproperty
+    def exposure_time(self):
+        """
+        The exposure time(s) of the observation(s).
+        """
+
+    ##### Misc. #####
+
+    @abc.abstractproperty
+    def rsun_meters(self):
+        """
+        Solar radius in units of length.
+        """
+
+    @abc.abstractproperty
+    def rsun_angular(self):
```

Comment:
Redundant given observer coordinate and `rsun_meters`?

Reply:
True.  But again in that case, this is still a useful convenience property.  Moreover who's to say that `rsun_meters` is provided in the header.  Perhaps only `rsun_angular` is.